src/test/regress/expected/multi_join_order_additional.out

```diff
@@ -77,66 +67,18 @@ CREATE TABLE customer_hash (
-DEBUG:  join prunable for intervals [1,1509] and [2951,4455]
-DEBUG:  join prunable for intervals [1,1509] and [4480,5986]
-DEBUG:  join prunable for intervals [1,1509] and [8997,10560]
-DEBUG:  join prunable for intervals [1,1509] and [10560,12036]
-DEBUG:  join prunable for intervals [1,1509] and [12036,13473]
-DEBUG:  join prunable for intervals [1,1509] and [13473,14947]
-DEBUG:  join prunable for intervals [1509,4964] and [8997,10560]
-DEBUG:  join prunable for intervals [1509,4964] and [10560,12036]
-DEBUG:  join prunable for intervals [1509,4964] and [12036,13473]
-DEBUG:  join prunable for intervals [1509,4964] and [13473,14947]
-DEBUG:  join prunable for intervals [2951,4455] and [1,1509]
-DEBUG:  join prunable for intervals [2951,4455] and [4480,5986]
-DEBUG:  join prunable for intervals [2951,4455] and [8997,10560]
-DEBUG:  join prunable for intervals [2951,4455] and [10560,12036]
-DEBUG:  join prunable for intervals [2951,4455] and [12036,13473]
-DEBUG:  join prunable for intervals [2951,4455] and [13473,14947]
-DEBUG:  join prunable for intervals [4480,5986] and [1,1509]
-DEBUG:  join prunable for intervals [4480,5986] and [2951,4455]
-DEBUG:  join prunable for intervals [4480,5986] and [8997,10560]
-DEBUG:  join prunable for intervals [4480,5986] and [10560,12036]
-DEBUG:  join prunable for intervals [4480,5986] and [12036,13473]
-DEBUG:  join prunable for intervals [4480,5986] and [13473,14947]
-DEBUG:  join prunable for intervals [8997,10560] and [1,1509]
-DEBUG:  join prunable for intervals [8997,10560] and [1509,4964]
-DEBUG:  join prunable for intervals [8997,10560] and [2951,4455]
-DEBUG:  join prunable for intervals [8997,10560] and [4480,5986]
-DEBUG:  join prunable for intervals [8997,10560] and [12036,13473]
-DEBUG:  join prunable for intervals [8997,10560] and [13473,14947]
-DEBUG:  join prunable for intervals [10560,12036] and [1,1509]
-DEBUG:  join prunable for intervals [10560,12036] and [1509,4964]
-DEBUG:  join prunable for intervals [10560,12036] and [2951,4455]
-DEBUG:  join prunable for intervals [10560,12036] and [4480,5986]
-DEBUG:  join prunable for intervals [10560,12036] and [13473,14947]
-DEBUG:  join prunable for intervals [12036,13473] and [1,1509]
-DEBUG:  join prunable for intervals [12036,13473] and [1509,4964]
-DEBUG:  join prunable for intervals [12036,13473] and [2951,4455]
-DEBUG:  join prunable for intervals [12036,13473] and [4480,5986]
-DEBUG:  join prunable for intervals [12036,13473] and [8997,10560]
-DEBUG:  join prunable for intervals [13473,14947] and [1,1509]
-DEBUG:  join prunable for intervals [13473,14947] and [1509,4964]
-DEBUG:  join prunable for intervals [13473,14947] and [2951,4455]
-DEBUG:  join prunable for intervals [13473,14947] and [4480,5986]
-DEBUG:  join prunable for intervals [13473,14947] and [8997,10560]
-DEBUG:  join prunable for intervals [13473,14947] and [10560,12036]
+LOG:  join order: [ "lineitem" ][ broadcast join "lineitem" ]
```

Comment:
How could this even be possible? `join order: [ "lineitem" ][ broadcast join "lineitem" ]`. How many shards `lineitem` have?

Reply:
lineitem has 2 shard, so it will be fixed by the following PR.